composer.json

```diff
@@ -22,11 +22,11 @@
         }
     ],
     "require": {
-        "php": ">=7.2.0",
+        "php": ">=8.0",
         "ext-intl": "*",
         "ext-json": "*",
         "ext-mbstring": "*",
-        "cakephp/chronos": "^2.0",
+        "cakephp/chronos": "^2.1.2",
```

Comment:
Should use 2.next for now to avoid errors on the PHP 8.1 build.

Reply:
Are you releasing 2.2. soon?

Reply:
Me personally no. I am no idea what else has gone into Chronos' next minor or what more needs to go. I just fixed the errors on PHP 8.1.